openedx/features/caliper_tracking/transformers/course_transformers.py

```diff
@@ -0,0 +1,15 @@
+"""
+Transformers for all course related events.
+"""
+
+
+def edx_course_enrollment_activated(current_event, caliper_event):
+    caliper_event['ip'] = current_event['ip']
+    if 'ip' in caliper_event['actor']:
+        del caliper_event['actor']['ip']
```

Comment:
Why is the deletion required? If the `ip` key is not required in `caliper_event` then it should not be there.

Reply:
It has been removed
As in the early version of master
base_transformer ip has been sent in actor's 
dict but now it's not